### ISSUE TITLE
Channel registration failed.

### ISSUE THREAD

**Issue by walsht** [urbanairship/ios-library#153]
Getting the below error when registering for push notifications today. Is this something on your side?
`[D] -[UAPush updateRegistration] [Line 762] APNS registration is out of date, updating.`
`[D] -[UAChannelRegistrar updateChannel:channelLocation:withPayload:] [Line 68] Updating channel c7644fxxxxxx`
`[D] -[UAChannelRegistrar updateChannel:channelLocation:withPayload:]_block_invoke [Line 82] Channel failed to update with JSON payload {"channel":{"opt_in":true,"locale_country":"US","set_tags":true,"ios":{"badge":0,"tz":"America\/Chicago"},"locale_language":"en","timezone":"America\/Chicago","background":true,"push_address":"xxxxxxx","tags":["Edition_US","Language_US"],"device_type":"ios"},"identity_hints":{"user_id":"xxxxx","device_id":"xxxxxxx"}}`
`[I] -[UAPush registrationFailedWithPayload:] [Line 933] Channel registration failed.`

**Comment by rlepinski** [comment]
@walsht Could you post verbose logs? it should have the error response.

**Comment by walsht** [comment]
Is there an email I can send the logs?

**Comment by crow** [comment]
mobileteam@urbanairship.com

**Comment by walsht** [comment]
Sent

**Comment by rlepinski** [comment]
@walsht We need verbose logging. You can set that in airship config under the developmentLogLevel. Set it to 5 in the plist or you can use UALogLevelTrace if you are doing it programatically.

**Comment by walsht** [comment]
Ok, will do.  I found when I go from 8.6.0 back to 8.5.3, it works correctly.

**Comment by walsht** [comment]
This is .what I am setting, let me know what I am missing:

`        let config = UAConfig.default()
        config.developmentAppKey = UAAppKey
        config.developmentAppSecret = UAAppSecret
        config.productionAppKey = UAAppKey
        config.productionAppSecret = UAAppSecret
        config.detectProvisioningMode = true
        UAirship.setLogging(true)
        UAirship.setLogLevel(.trace)
        UAirship.takeOff(config)`

**Comment by rlepinski** [comment]
Yeah its a little confusing. Setting the log level on UAirship directly will get overridden by UAConfig. Do `config.developmentLogLevel = .trace`

**Comment by walsht** [comment]
Sent

**Comment by rlepinski** [comment]
Alright, we traced it down to the CRA payload change that we made in 8.6 to populate some OOTB tags quicker. The iOS subsection of the payload use to only contain a timezone if quiet time was set, but thats no longer the case in 8.6. We did not catch this locally because the only way this manifests itself is if we have auto badging enabled. Working with our platform team to be more lenient with the payload but we may have to do a patch release. Ill report back when its fixed.
